specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/networkWatcher.json

```diff
@@ -4288,6 +4352,10 @@
         "disableTraceRoute": {
           "type": "boolean",
           "description": "Value indicating whether path evaluation with trace route should be disabled."
+        },
+        "destinationPortBehavior": {
```

Comment:
what are the allowed values here?  Can it be a string enum so the user knows what can be entered in this property?